Demo/Application/Features/PayPalWebCheckoutViewController.swift

```diff
@@ -95,6 +95,11 @@ class PayPalWebCheckoutViewController: PaymentButtonBaseViewController {
             action: #selector(tappedPayPalAppSwitchForCheckout)
         )
         
+        let payPalAppSwitchForCreditButton = createButton(
```

Comment:
We may need to bump the `heightConstraint` in the `viewDidLoad` so the new button isn't quite so squished